### DISCUSSION TITLE
Publishing quarto presentation to quarto-pub

### DISCUSSION THREAD

**Discussion by lervin1** [quarto-dev/quarto-cli#6631] [Q&A]
### Description

Hi, I am having trouble publishing my quarto presentation to quarto pub...
![screenshot1](https://github.com/quarto-dev/quarto-cli/assets/143349771/c8f596b3-89cb-43af-9a7a-3a8b88be58b0)
This is what my error looks like from the terminal.

**Comment by mcanouil** [comment]
See #6624
